src/scripts/star-rating.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+var star1 = document.getElementsByClassName('star-1');
+var star2 = document.getElementsByClassName('star-2');
+var star3 = document.getElementsByClassName('star-3');
+var star4 = document.getElementsByClassName('star-4');
+var star5 = document.getElementsByClassName('star-5');
+
+// Function that marks first star //
+
+const firstStar = function() {
+  for(let i = 0; star1.length > i; i++){
```

Comment:
This loop definition is totally fine, but you may notice that `for` loops in 99% are formed in this way:
```
for (let i = 0; i < array.length; i++) {
  …
}
```
It's kind of convention. So next time I would rather use `i < array.length` instead it's inverted form `array.length > i`.